src/components/Calculator.js

```diff
@@ -1,76 +1,180 @@
 import React from 'react';
 import './Calculator.css';
+import calculate from '../logic/calculator';
```

Comment:
- [ ] Both logic scripts `calculator.js & operate.js` could be imported on this component, it looks like you do it in the other logic script, please, add `operate.js` here. 🐶

Reply:
![image](https://user-images.githubusercontent.com/79217155/129250120-91447520-3f54-4c97-9a29-a9f5b77f7cfd.png)

The operate function is only needed in the calculate function it is unnecessary to use the operate function in the calculator itself as it has no use...